src/validation/ValidationSchema.js

```diff
@@ -0,0 +1,55 @@
+import * as yup from 'yup';
+import AmountType from '../types/AmountType';
+
+/* eslint-disable import/prefer-default-export */
+export const validationSchema = yup.object({
+  name: yup.string().required('Please enter the scholarship name'),
+  deadline: yup
+    .date()
+    .required('Please enter the scholarship deadline')
+    .typeError('Please enter a valid date'),
+  description: yup
+    .string()
+    .required('Please enter the scholarship description'),
+  website: yup
+    .string()
+    .url('Website must be a valid URL')
+    .required('Please enter the scholarship website'),
+  amount: yup.object().shape({
+    type: yup
+      .mixed()
+      .oneOf(Object.values(AmountType), 'Please choose an option above'),
+    minAmount: yup
+      .number()
+      .when('type', {
+        is: AmountType.Fixed,
+        then: yup
+          .number()
+          .required()
+          .moreThan(0, 'Please enter the scholarship amount'),
+      })
+      /* eslint-disable func-names */
+      .test('test', 'Minimum must be less than the maximum', function (value) {
```

Comment:
I think you can fix the lint issue (here and elsewhere) by using arrow functions:
```jsx
      .test('test', 'Minimum must be less than the maximum', (value) => {
```

Reply:
But then `this` returns undefined for `{type, maxAmount}`

Reply:
Ah, maybe this then?
```jsx
      .test('test', 'Minimum must be less than the maximum', function test(value) {
```

Or you can leave it as is too